src/dlr_tensorflow/dlr_tensorflow.cc

```diff
@@ -343,16 +343,6 @@ TensorflowModel::TensorflowModel(
   }
   TF_DeleteSessionOptions(opts);
   LOG(INFO) << "Tensorflow Session was created";
-
-  // Run inference to allocate output Tensors and calculate output shapes.
-  for (int i = 0; i < num_inputs_; i++) {
-    TF_Tensor* tensor = input_tensors_[i];
-    int64_t num_elements = TF_TensorElementCount(tensor);
-    float* in_t_data = (float*)TF_TensorData(tensor);
-    std::fill_n(in_t_data, num_elements, 0.1);
-  }
-  Run();
-  LOG(INFO) << "Output Tensors were allocated";
```

Comment:
Methods which depends on inference result such as `GetDLROutputSizeDim`, `GetDLROutputShape` should be called after `RunDLRModel`.  Running first "warmup" inference inside TensorflowModel constructor does not always work because output shape might depend on input data. E.g. Style transform model output Height and Width depends on input image Height and Width. Removing this block was my old tech debt.